src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import './App.css';
+import PropTypes from 'prop-types';
 
-const App = () => (
-  <h1>React sum</h1>
+const App = ({ a, b }) => (
```

Comment:
```suggestion
const App = props => (
```
In React, you pass some data to component with `props`

Reply:
I am well aware of that :) But it's destructuring assignment and as far as I know it's ok to use it. Also, when you have to use some property multiple times it's just more convenient to write `a` rather than `props.a`

Reply:
Alright, maybe you are right 👍

Reply:
Yaroslav, you are right. Greate job.